sfa_dash/templates/head.html

```diff
@@ -35,14 +35,22 @@
 <script src="https://cdn.pydata.org/bokeh/release/bokeh-tables-{{ bkversion }}.min.js"></script>
 {% endif %}
 
+{# internal script inclusions #}
 <script src="/static/js/table-search.js"></script>
 <script src="/static/js/table-checkall.js"></script>
-<script src="https://cdn.plot.ly/plotly-1.52.3.min.js"></script>
-{# block injecting global variables from app config, passed through
-template_globals #}
+
+{# block injecting app config to the global sfa_dash_config object, passed
+   through template_globals #}
 <script>
-    var MAX_DATA_RANGE_DAYS = {{ MAX_DATA_RANGE_DAYS }};
-    var MAX_PLOT_DATAPOINTS = {{ MAX_PLOT_DATAPOINTS }};
+    var sfa_dash_config = {
+      MAX_DATA_RANGE_DAYS: {{ MAX_DATA_RANGE_DAYS }},
+      MAX_PLOT_DATAPOINTS: {{ MAX_PLOT_DATAPOINTS }},
+      VARIABLE_NAMES: JSON.parse('{{ variable_names | tojson }}'),
+      VARIABLE_UNIT_MAP: JSON.parse('{{ variable_unit_map | tojson }}'),
+      METRIC_CATEGORIES: JSON.parse('{{ metric_categories | tojson }}'),
+      DETERMINISTIC_METRICS: JSON.parse('{{ deterministic_metrics | tojson }}'),
+      ALL_METRICS: JSON.parse('{{ all_metrics | tojson }}'),
+    }
```

Comment:
@wholmgren, I added a  global `sfa_dash_config` variable that gets loaded in the <head> element, and we can load python dicts into it with the `tojson` filter. What do you think of this? There are still common functions that could be pulled into a module, but this will help with the configuration we want to pull in from core.

Reply:
sounds reasonable to me